zipline/pipeline/filters/filter.py

```diff
@@ -488,3 +489,18 @@ def _compute(self, arrays, dates, assets, mask):
                 asset=self._asset, start_date=dates[0], end_date=dates[-1],
             )
         return out
+
+
+class SmoothingFilter(CustomFilter):
+    """
+    A Filter that requires its inputs to have
+    been True for the last `window_length` days.
+    An integral part of the Q500US methodology
```

Comment:
This note doesn't belong here.  A reader of Zipline won't necessarily know what Q500US is, and nothing about the behavior here is specific to that project.